indico/modules/events/surveys/util.py

```diff
@@ -59,3 +65,35 @@ def was_survey_submitted(survey):
     if submission_id is None:
         return False
     return bool(SurveySubmission.find(id=submission_id).count())
+
+
+class CSVSubmissionsExporter(object):
+
+    BASE_FIELDNAMES = {'submitter', 'submission_date'}
+
+    def __init__(self, survey):
+        self._survey = survey
+        self._fieldnames = self.BASE_FIELDNAMES | {question.title for question in self._survey.questions
+                                                   if not isinstance(question.field, StaticTextField)}
+
+    def generate(self):
+        with NamedTemporaryFile(suffix='indico.tmp', dir=Config.getInstance().getTempDir(), delete=False) as temp_file:
+            writer = csv.DictWriter(temp_file, fieldnames=self._fieldnames)
+            writer.writeheader()
+            for submission in self._survey.submissions:
+                submission_dict = {
+                    'submitter': (encode_if_unicode(to_unicode(submission.user.full_name))
```

Comment:
REALLY? the csv writer doesn't handle unicode strings?!?!?!

anyway, `submission.user.full_name` is guaranteed to be unicode, so you can `submission.user.full_name.encode('utf-8')`